src/App.tsx

```diff
@@ -9,24 +9,28 @@ import * as api from "./api";
 import Filemanager from "./Filemanager";
 import numeral from "numeral";
 
-const player = new WSAvcPlayer({ useWorker: false });
+const player = new WSAvcPlayer({ useWorker: false }); // declaration of the video player
 const retryInterval = 1000;
 const iconSize = 32;
 
 export interface PhotoMode {
+  // implement when you are in photo mode or not
   photoMode: boolean;
   setPhotoMode: (value: boolean) => void;
 }
```

Comment:
```suggestion
// photo mode state (will take a picture instead of recording a video)
export interface PhotoMode {
  photoMode: boolean;
  setPhotoMode: (value: boolean) => void;
}
```